sweep/sweeper.go

```diff
@@ -631,12 +631,6 @@ func (s *UtxoSweeper) collector(blockEpochs <-chan *chainntnfs.BlockEpoch) {
 		return
 	}
 
-	// Create a ticker based on the config duration.
-	ticker := time.NewTicker(s.cfg.TickerDuration)
```

Comment:
What about the integration tests? We use this in many tests to force quick sweeps w/o mining blocks, as we may want to test the case where they're all mined in a single block.

Reply:
Yes and this makes the test easier as it's now deterministic - we can always use a block to trigger the sweeping behavior, without waiting for a few milliseconds. And they are still mined in the same block as the sweeper can accept new sweep inputs at any time, it's just the actual sweeping tx won't be created and published unless a new block comes in.

BUT...there are quite some itests need to be updated tho, still working through...